kiwi/cli.py

```diff
@@ -119,7 +125,9 @@ def get_servicename(self):
         :return: service name
         :rtype: string
         """
-        if self.all_args['system']:
+        if self.all_args['image']:
+            return 'image'
```

Comment:
Scary. :laughing: It is generally a bad idea to hammer the key directly on _testing_ it. I would prefer:
```python
if self.all_args.get('anykey') is not None:
    return 'whatever needs to be here'
elif ...
```